src/libraries/sendtohelixhelp.proj

```diff
@@ -284,6 +284,53 @@
       <HelixCorrelationPayload Condition="$(TargetRuntimeIdentifier.ToLowerInvariant().StartsWith('linux-bionic'))" Include="openssl" Uri="https://netcorenativeassets.blob.core.windows.net/resource-packages/external/android/openssl-1.1.1l-beta-1.zip" Destination="openssl" />
     </ItemGroup>
 
+    <PropertyGroup>
+      <IsTargetOSSupportedByXUnitLogChecker Condition="'$(TargetOS)' != 'browser' and '$(TargetOS)' != 'wasi' and '$(TargetOS)' != 'ios' and '$(TargetOS)' != 'iossimulator' and '$(TargetOS)' != 'tvos' and '$(TargetOS)' != 'tvossimulator' and '$(TargetOS)' != 'maccatalyst' and '$(TargetOS)' != 'android'">true</IsTargetOSSupportedByXUnitLogChecker>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(IsTargetOSSupportedByXUnitLogChecker)' == 'true'">
+      <XUnitLogCheckerDirectory>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)tests/Common/XUnitLogChecker'))</XUnitLogCheckerDirectory>
+      <XUnitLogCheckerDumpsPath Condition="'$(TargetOS)' != 'windows'">%24HELIX_DUMP_FOLDER</XUnitLogCheckerDumpsPath>
+      <XUnitLogCheckerDumpsPath Condition="'$(TargetOS)' == 'windows'">%25HELIX_DUMP_FOLDER%25</XUnitLogCheckerDumpsPath>
+      <XUnitLogCheckerExecutable Condition="'$(TargetOS)' != 'windows'">./$(XUnitLogCheckerDirectory)XUnitLogChecker</XUnitLogCheckerExecutable>
+      <XUnitLogCheckerExecutable Condition="'$(TargetOS)' == 'windows'">$(XUnitLogCheckerDirectory)XUnitLogChecker.exe</XUnitLogCheckerExecutable>
+      <XUnitLogCheckerCommand>$(XUnitLogCheckerExecutable) --dumps-path $(XUnitLogCheckerDumpsPath)</XUnitLogCheckerCommand>
```

Comment:
One thing that we need to be careful about is calling the EXE directly might use the host's default framework probing.  That's fine if you expect a machine-wide framework on the machine that the host can use - or environment variables are set to tell the host where to look.  I'm not sure if that's the case.

If you need to use the same `dotnet` that the test is using you can use `%HELIX_CORRELATION_PAYLOAD%\dotnet`
https://github.com/dotnet/runtime/pull/93906/files#diff-1d68a522b60c1ab69fb537ce06f5d7e9a176fa4f75e16425fc164d4b738d7678L220

It's unclear to me if that's what we want - since that might be an unstable runtime.  I wonder what CLR tests do - do they use the runtime under development or do they deploy an LKG runtime to run the tool on?